app/views/events/show.html.erb

```diff
@@ -66,9 +67,26 @@
         <i class="fa fa-calendar"></i> Add to Google Calendar
       <% end %>
     <% end %>
-        
+
+  </div>
+  
+  <div class="share-container">
+    <% unless @event.end_date.past? %>  
+      <%= rrssb_tag(url:"https://css.uwindsor.ca/" + request.original_fullpath , buttons: %w{facebook twitter google_plus email linkedin}) %>
+    <% end %>
   </div>
+
+  <script type="text/javascript">
+
+    jQuery(document).ready(function () {
```

Comment:
This is still incomplete. I was not sure if this is the correct way to change the buttons styling?